sdk/containerregistry/azure-containerregistry/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Bugs Fixed
 
 ### Other Changes
+* Change `audience` to an optional parameter in `ContainerRegistryClient`, which the default value is `"https://management.azure.com"`. ([#27675](https://github.com/Azure/azure-sdk-for-python/pull/27675))
```

Comment:
I wouldn't put the pull request in the changelog - though you could put the issue number.

Reply:
Sounds good, I'll make the change.